src/core/image.js

```diff
@@ -847,6 +886,9 @@ class PDFImage {
     }
 
     imgData.data = data;
+    if (isOffscreenCanvasSupported && mustBeResized) {
```

Comment:
Nit: Given the definition it looks like the following would be sufficient.
```suggestion
    if (mustBeResized) {
```